src/main/java/org/reactome/release/qa/Main.java

```diff
@@ -99,8 +99,8 @@ public static void main(String[] args) throws Exception {
             }
             else {
                 String fileName = check.getDisplayName();
-                qaReport.output(fileName + ".txt", output.getAbsolutePath());
-                logger.info("Check "+ output.getAbsolutePath() + "/" + fileName + ".txt for report details.");
+                qaReport.output(fileName + ".tsv", output.getAbsolutePath());
```

Comment:
Be aware: changing file extension may break html links generated in the weekly report. Please make sure.

Reply:
Here it is an example: https://curator.reactome.org/QAReports/20200525/ReleaseQA/Attribute_Has_Only_One_Value_wug.html. The html file was generated based on these output files.

Reply:
I'll revert it, my mistake